node-require/calculate.js

```diff
@@ -0,0 +1,17 @@
+const add = require('./add');
+const subtract = require('./subtract');
+const multiply = require('./multiply');
+const divide = require('./divide');
+
+const numberX = Math.floor(process.argv[2]);
+const numberY = Math.floor(process.argv[4]);
```

Comment:
The calculator should still produce accurate output when the inputs contain decimal places.